src/filesystem.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#define IS_IRALL (S_IRUSR | S_IRGRP | S_IROTH) /* read permission, all */
+#define IS_IWALL (S_IWUSR | S_IWGRP | S_IWOTH) /* write permission, all */
+#define IS_IXALL (S_IXUSR | S_IXGRP | S_IXOTH) /* execute/search permission, all */
```

Comment:
It's for calling `chmod` with read/write/execute permission for everybody. FatFS posix implementation uses it as FAT doesn't have user specific permissions. I used it also during testing, so I thought it can be available for everybody.

Reply:
Ok, can you, please, add some docstring to those?

Reply:
Done